Source/player.h

```diff
@@ -7,8 +7,27 @@
 
 #include <stdint.h>
 
+#include "gendung.h"
+#include "items.h"
+#include "spelldat.h"

```

Comment:
Ok I think I see the pattern now. Is it a best practice to isolate `<x>` includes from `"x"` ones?